app/views/ai/HackstackAutoPromotion.vue

```diff
@@ -0,0 +1,31 @@
+<script>
+import ModalDynamicPromotion from 'ozaria/site/components/teacher-dashboard/modals/ModalDynamicContent'
+import HackstackPromotionComponent from './HackstackPromotionComponent'
+export default {
+  components: {
+    ModalDynamicPromotion,
+    HackstackPromotionComponent,
+  },
+  methods: {
+    openModalDynamicPromotion () {
```

Comment:
what is this method for?

Reply:
thank you for find out some unused function